OneBlink.SDK.Tests/OneBlink.SDK.Tests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
Should this be in the changelog or does it only effect what we test against?

Reply:
Only effects what we run the tests agaisnt, the SDK is still support .Net Standard 2.0